libs/langchain/langchain/memory/chat_memory.py

```diff
@@ -26,9 +26,12 @@ def _get_input_output(
         else:
             prompt_input_key = self.input_key
         if self.output_key is None:
-            if len(outputs) != 1:
-                raise ValueError(f"One output key expected, got {outputs.keys()}")
-            output_key = list(outputs.keys())[0]
+            if len(outputs) == 1:
+                output_key = list(outputs.keys())[0]
+            elif "output" in outputs:
+                output_key = "output"
+            else:
+                raise ValueError("Could not determine output key.")
```

Comment:
can we raise a better error message? should contain info about the keys found and why we cant determine the output key

Reply:
how about changing it to `ValueError(f"Got multiple output keys: {outputs.keys()}, cannot determine which to store in memory. Please set the 'output_key' explicitly.")`